features/promotion/receiving_discount/receiving_discounts_with_product_minimum_price_specified.feature

```diff
@@ -0,0 +1,73 @@
+@receiving_discount
+Feature: Receiving discounts with product minimum price specified
+    In order to pay avoid paying less than product minimum price
+    As a Visitor
+    I want to receive discount for my purchase up to product minimum price
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store classifies its products as "T-Shirts"
+        And the store has a "T-Shirt" configurable product
+        And this product has "PHP T-Shirt" variant priced at "$50.00"
+        And the "PHP T-Shirt" variant has minimum price "$45.00" in "United States" channel
+        And the store has a product "PHP Mug" priced at "$20.00"
+        And there is a promotion "Christmas sale"
+
+    @ui @api
+    Scenario: Receiving percentage discount on a single item fulfilling minimum price criteria
+        Given this promotion gives "50%" off on every product with minimum price at "$50.00"
```

Comment:
I know that is minor thing but, can you use here different promotion rule. why? because `minimum price of product in cart` is a bit misleading in behats about `minimum price for variants` (but different one :D)

Reply:
thats true, but this way we can prove that promotion takes it to account before calculating minimum price

Reply:
added these scenarios along with different prices so that we can track every cent of product's price